packages/compass-crud/src/components/expand-all-documents-button.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {
+  Icon,
+  Button,
+  css,
+  WorkspaceContainer,
+} from '@mongodb-js/compass-components';
+
+type ExpandAllDocumentsButtonProps = {
+  allDocumentsExpanded: boolean;
+  onClick: () => void;
+};
+
+const hiddenOnNarrowStyles = css({
+  [`@container ${WorkspaceContainer.toolbarContainerQueryName} (width < 900px)`]:
+    {
+      display: 'none',
+    },
+});
+
+const ExpandAllDocumentsButton: React.FunctionComponent<
+  ExpandAllDocumentsButtonProps
+> = ({ allDocumentsExpanded, onClick }) => {
+  const buttonText = allDocumentsExpanded ? 'Collapse' : 'Expand';
+
+  return (
+    <Button
+      value={`${buttonText} all documents`}
+      size="xsmall"
+      onClick={onClick}
+      leftGlyph={<Icon glyph="Edit"></Icon>}
```

Comment:
I am not sure, what icon should we use.

Reply:
I added a comment below related to this.